nuxeo-connect-client/src/main/java/org/nuxeo/connect/packages/PackageManagerImpl.java

```diff
@@ -1036,7 +1036,9 @@ private void orderByDependencies(Map<String, DownloadablePackage> allPackagesByI
                 }
                 listToOrder.clear();
             } else {
-                throw new DependencyException(String.format("Couldn't order %s missing %s.", listToOrder, missingDeps));
+                throw new DependencyException(String.format(
+                        "Couldn't order %s missing %s (you may consider to relax or allow snapshots).", listToOrder,
```

Comment:
`consider to` is bad English. Maybe instead say `consider using --relax true or --snapshot`?